src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -33,6 +35,7 @@
 import java.util.Set;
 
 public class RestSqlAction extends BaseRestHandler {
+    public static Logger logger = LogManager.getLogger(RestSqlAction.class);
```

Comment:
Might be better to declare this Logger ```private static final```

Reply:
Oops. Good catch. Updated.

Reply:
Thanks for the change! Mohammad was referring to the logger convention like this: https://github.com/opendistro-for-elasticsearch/sql/blob/c126e456538167283df92f49850aa93e0307e69a/src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java#L44

I will try to fix the NPE issue this week.

Reply:
I have updated with package convention in third commit.